components/src/core/components/TinyMce/TinyMce.vue

```diff
@@ -154,6 +156,43 @@ export default defineComponent({
           };
           reader.onerror = (error: any) => {};
         };
+
+        setTinymceBackGroundImage.value = (file: File) => {
+          imageUploadForBackgound.value = false;
+          const reader = new FileReader();
+          reader.readAsDataURL(file);
```

Comment:
How will this check the uploaded file is of approved types or size

Reply:
it is using the currently used validations for image upload